### DISCUSSION TITLE
Is it possible for windows users to enable JIT on iOS 17 by installing macos on windows on virtual box ?

### DISCUSSION THREAD

**Discussion by abhishekgi** [SideStore/SideStore#527] [Q&A]
Is it possible for windows users to enable JIT on iOS 17 by installing macos on windows on virtual box ?

**Comment by amsam0** [comment]
Probably not, but you could try

**Comment by abhishekgi** [comment]
Hmm

**Comment by polymo1** [comment]
Please consider using SideJITServer instead ( https://github.com/nythepegasus/sidejitserver ).
